### PR TITLE
[Serve] Fix the serve.batch api doc

### PR DESCRIPTION
**Pull request by sihanwang41**
## Why are these changes needed?

Current version:
image

Fixed version:
image

## Checks

- [ ] I've signed off every commit(by using the -s flag, i.e., `git commit -s`) in this PR.
- [ ] I've run `scripts/format.sh` to lint the changes in this PR.
- [ ] I've included any doc changes needed for https://docs.ray.io/en/master/.
    - [ ] I've added any new APIs to the API Reference. For example, if I added a 
           method in Tune, I've added it in `doc/source/tune/api/` under the 
           corresponding `.rst` file.
- [ ] I've made sure the tests are passing. Note that there might be a few flaky tests, see the recent failures at https://flakey-tests.ray.io/
- Testing Strategy
   - [ ] Unit tests
   - [ ] Release tests
   - [ ] This PR is not tested :(

### CODE REVIEW

**Review by edoakes — APPROVED** [review]
Nice, is there a way to have our linter or something auto-detect this?

File: python/ray/serve/batching.py [hunk 1]
```diff
@@ -228,14 +228,13 @@ def batch(_func=None, max_batch_size=10, batch_wait_timeout_s=0.0):
...
-    ... async def handle_batch(batch: List[str]): # doctest: +SKIP
-    ...     return [s.lower() for s in batch] # doctest: +SKIP
-
-    >>> async def handle_single(s: str): # doctest: +SKIP
-    ...     # Returns s.lower().
-    ...     return await handle_batch(s) # doctest: +SKIP
+        >>> from ray import serve
+        >>> @serve.batch(max_batch_size=50, batch_wait_timeout_s=0.5) # doctest: +SKIP
+        ... async def handle_batch(batch: List[str]): # doctest: +SKIP
+        ...     return [s.lowerer() for s in batch] # doctest: +SKIP
```
Comment (shrekris-anyscale) [inline python/ray/serve/batching.py:234]:
```suggestion
        ...     return [s.lower() for s in batch] # doctest: +SKIP
```

**Review by shrekris-anyscale — APPROVED** [review]
Nice, left a comment.

**Comment by sihanwang41** [comment]
> Nice, is there a way to have our linter or something auto-detect this?
I don't know actually, @maxpumperla if you have insights...

**Comment by architkulkarni** [comment]
//doc:source/ray-air/doc_code/computer_vision        failure unrelated
Windows test_task_events failure unrelated

**Pull request merged**